class ExceptionDemo2 
{

    static
	{
		try
		{
			throw new ArithmeticException();
		}
		catch (ArithmeticException e)
		{
		}
	}

	public static void main(String[] args) 
	{
		
	}
}